src/app/modules/form/admin-form/__tests__/admin-form.controller.spec.ts

```diff
@@ -9798,4 +9801,139 @@ describe('admin-form.controller', () => {
       expect(MockAdminFormService.getFormField).not.toHaveBeenCalled()
     })
   })
+
+  describe('handleGetFreeSmsCountForFormAdmin', () => {
+    const mockForm = {
+      admin: new ObjectId().toHexString(),
+    } as unknown as IFormSchema
+    const VERIFICATION_SMS_COUNT = 3
+
+    beforeAll(() => {
+      MockFormService.retrieveFormById.mockReturnValue(okAsync(mockForm))
+      MockSmsService.retrieveFreeSmsCounts.mockReturnValue(
+        okAsync(VERIFICATION_SMS_COUNT),
+      )
+    })
+
+    it('should retrieve counts and msgSrvcId when the user and the form exist', async () => {
+      // Arrange
+      const MOCK_REQ = expressHandler.mockRequest({
```

Comment:
nit: if its not a globally defined constant, this should be in camelCase.

Same applies for the rest of invocations in this test